Source/WebKit/UIProcess/API/Cocoa/_WKApplicationManifest.h

```diff
@@ -72,7 +73,9 @@ WK_CLASS_AVAILABLE(macos(10.13.4), ios(11.3))
 @property (nonatomic, readonly, copy) NSURL *startURL;
 @property (nonatomic, readonly, copy) NSURL *manifestId WK_API_AVAILABLE(macos(13.3), ios(16.4));
 @property (nonatomic, readonly) _WKApplicationManifestDisplayMode displayMode;
+@property (nonatomic, readonly, copy) NSArray<NSString *> *categories WK_API_AVAILABLE(macos(14.2), ios(17.2));
```

Comment:
Please use WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA))

Reply:
Switched to `WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA))` for both.